src/main/java/net/dv8tion/jda/api/entities/MessageChannel.java

```diff
@@ -2065,10 +2061,7 @@ default MessagePaginationAction getIterableHistory()
      *         The unicode characters of the emoji
      *
      * @throws java.lang.IllegalArgumentException
-     *         <ul>
-     *             <li>If provided {@code messageId} is {@code null} or empty.</li>
-     *             <li>If provided {@code unicode} is {@code null} or empty.</li>
-     *         </ul>
+     *         If provided {@code unicode} is {@code null} or empty.
```

Comment:
Why did this lose the bullet point about `messageId` ?

Reply:
Since this is the `long` overload, null or something empty cannot be passed as `messageId`. However, I misunderstood the `Checks.isSnowflake` method and this can still throw if `messageId` is not a valid snowflake. I added a bullet point about that situation now.